openpype/hosts/houdini/plugins/create/create_vray_rop.py

```diff
@@ -54,6 +57,15 @@ def create(self, subset_name, instance_data, pre_create_data):
             "SettingsEXR_bits_per_channel": "16"   # half precision
         }
 
+        if pre_create_data.get("export_job"):
+            scene_filepath = "{export_dir}{subset_name}/{subset_name}.$F4.vrscene".format(
+                export_dir=hou.text.expandString("$HIP/pyblish/vrscene/"),
+                subset_name=subset_name,
+            )
+            # TODO: don't have VRay to check the names of these
```

Comment:
@BigRoy do you have a VRay license to check this?

Reply:
Hmm - maybe only an older v-ray license unfortunately. Don't think it'll be a compatible version to recent Houdini version so I assume no.